### DISCUSSION TITLE
hiPSC cell activity detected as background noise

### DISCUSSION THREAD

**Discussion by JalterMain** [flatironinstitute/CaImAn#1234] [Q&A]
Hello, I'm currently working on a project using caiman to detect ROIs from hiPSC cell calcium imaging. However, a recurring problem is that a lot of the activity is detected as background (see attached, first image is background and second image is the ROI detection). I tried adjusting the parameters for component evaluation, but the results have been mixed. Is this due to the synchronous nature of hiPSC cells or any other possible reason?
![1](https://github.com/flatironinstitute/CaImAn/assets/68760823/620d456d-8eb6-4bc1-b741-8f6e2d2b1d20)
![2](https://github.com/flatironinstitute/CaImAn/assets/68760823/440b6516-66b0-47d8-8777-7270268e69e4)

**Comment by EricThomson** [comment]
Thanks for the question it sounds like a cool data set! Those images are not showing for me, not sure why.

I'd need to see more details like a little sample movie (you can send it to me if it can't be made public), and I'd like to know more details about the analysis you are running.

Note things are a little slow right now because of the SFN-Thanksgiving holiday in the US.

**Reply by JalterMain** [reply]
Thanks Eric! I've sent you via email a few samples movies (tif files) and associated images. As for the study, I can't divulge all the details (nor do I know all of them since I'm working on analysis), but I can say that we are using caiman to study the difference between treated vs untreated hiPSC cells. We use caiman to extract the ROIs and all the associated data, and from there we analyze the differences in terms of frequency, duration, and synchronicity. The pipeline we are running is similar to demo_pipeline.py file, with a few modifications here and there, such as cutting out the first 35 frames of the movie due to changing background levels. Thank you!

**Comment by JalterMain** [comment]
Hello,
This is JalterMain from Github Q&A hiPSC cell activity as background noise
#1234. I've attached the images that were not loading earlier on my
previous github post as well as a few sample movies.
 A1-1.tif
<https://drive.google.com/file/d/10LmEAKlTAXYbBwe94ItZ-MUAG-geays-/view?usp=drive_web>

 A2.tif
<https://drive.google.com/file/d/1yFo-o5Qdcle7DKfrI2ltD8fVwTKdvVRw/view?usp=drive_web>
Have a great day,
Matthew

**Reply by EricThomson** [reply]
Thanks sorry for the delay with holiday and other things it may be a little while before I can check this out:. Please ping me if I don't get to this soon (e.g., if I haven't looked at this next week).
